excalidraw-app/App.tsx

```diff
@@ -630,6 +654,43 @@ const ExcalidrawWrapper = () => {
   ) => {
     if (collabAPI?.isCollaborating()) {
       collabAPI.syncElements(elements);
+    } else if (excalidrawAPI) {
+      const currentReminderIndex =
+        reminderState && reminderState.sceneId === appState.id
+          ? reminderState.nextIndex
+          : 0;
+      const now = Date.now();
+      // if delta is less than first threshold, then we recently saved
+      if (now - appState.lastSave.timestamp < REMINDER_THRESHOLDS[0].time) {
+        if (currentReminderIndex !== 0) {
+          updateReminderState({
+            nextIndex: 0,
+            sceneId: appState.id,
+          });
+        }
+      } else if (currentReminderIndex < REMINDER_THRESHOLDS.length) {
+        const reminderThreshold = REMINDER_THRESHOLDS[currentReminderIndex];
+        const nonDeletedElements = getNonDeletedElements(elements);
+        // Remind if we haven't saved for too long
+        if (
+          now - appState.lastSave.timestamp >= reminderThreshold.time &&
+          nonDeletedElements.length - appState.lastSave.elementsCount >=
+            reminderThreshold.elementsCount &&
```

Comment:
I'd do both `(time && count)` as is now, but also trigger on `count` alone so that if user creates tons of elements fast, we still tell them to save, because they could then leave and come back after two weeks after they clear their site data (or the browser does it for them).

But we may set different element count thresholds for the "count" condition alone.